content/docs/2.9/scalers/loki.md

```diff
@@ -0,0 +1,165 @@
+go_file = "loki_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `loki` trigger that scales based on a Loki query result. Here is an example of providing values in metadata:
+
+```yaml
+triggers:
+- type: loki
+  metadata:
+    # Required fields:
+    serverAddress: http://<loki-host>:3100 # Note: loki server URL 
+    metricName: syslog_write_total # Note: name to identify the metric, generated value would be `loki-syslog_write_total`
+    query: sum(rate({filename="/var/log/syslog"}[1m])) by (level) # Note: query must return a vector/scalar single element response
+    threshold: '0.7'
+    # Optional fields:
+    activationThreshold: '2.50'
+    tenantName: Tenant1 # Optional. X-Scope-OrgID header for specifying the tenant name in a multi-tenant setup.
+    ignoreNullValues: false # Default is `true`, which means ignoring the empty value list from Loki. Set to `false` the scaler will return error when Loki target is lost
+    unsafeSsl: false #  Default is `false`, Used for skipping certificate check when having self signed certs for Loki endpoint
+```
+
+**Parameter list:**
+
+- `serverAddress` - URL of Loki server.
+- `metricName` - Name to identify the metric in the external.metrics.k8s.io API. If using more than one trigger it is required that all `metricName`(s) be unique.
+- `query` - LogQL query to run. The query must return a vector/scalar single element response.
+- `threshold` - Value to start scaling for. (This value can be a float)
+- `activationThreshold` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional, This value can be a float)
+- `tenantName` - The `X-Scope-OrgID` header for specifying the tenant name in a multi-tenant setup. (Optional)
+- `ignoreNullValues` - Value to reporting error when Loki target is lost. (Values: `true`,`false`, Default: `true`, Optional)
+- `unsafeSsl` - Used for skipping certificate check e.g: using self signed certs. (Values: `true`,`false`, Default: `false`, Optional)
+- `authMode` - Authentaication mode to be used. (Values: `bearer`,`basic`, Optional)
+
+### Authentication Parameters
+
+Loki doesn't provide any kind of authentication out of the box. However, most commonly Loki is configured along with a Basic Auth or Bearer Auth, which are the only valid options for authentication here.
+
+You can use `TriggerAuthentication` CRD to configure the authentication. Specify `authModes` and other trigger parameters along with secret credentials in `TriggerAuthentication` as mentioned below:
+
+**Bearer authentication:**
+- `authModes`: It must contain `bearer` in case of Bearer Authentication. Specify this in trigger configuration.
+- `bearerToken`: The token needed for authentication.
+
+**Basic authentication:**
+- `authMode`: It must contain `basic` in case of Basic Authentication. Specify this in trigger configuration.
```

Comment:
```suggestion
- `authModes`: It must contain `basic` in case of Basic Authentication. Specify this in trigger configuration.
```
This should be `authModes` as per the parsing method used in the code.